doc/devbox_setup.md

```diff
@@ -0,0 +1,115 @@
+# Prepare your development environment
+
+This document describes how to prepare your development environment to build and use the **Microsoft Azure IoT device client SDK for .NET**
+
+1.  [Prerequisites required to build the SDK](#min_setup)
+2.  [Optional prerequisites required to test Xamarin, Windows IoT and .NET Micro Framework](#advanced)
+3.  [Test prerequisites](#testprereq)
+
+<a name="min_setup"/>
+
+## Prerequisites required to build the SDK
+
+The following is the minimum required to build and produce IoT SDK binaries. 
+This may not be sufficient to run certain samples that require Visual Studio and other SDKs or frameworks installed on your machine.
+
+### Windows + .NET Framework
+
+- Install the latest .NET Core from https://dot.net
+- Install .NET Framework 4.7 Developer Pack: https://support.microsoft.com/en-us/help/3186612/the-net-framework-4-7-developer-pack-and-language-packs
+- Install .NET Framework 4.5.1 Developer Pack: https://www.microsoft.com/en-us/download/details.aspx?id=40772
+- As admin (one-time setup):
+    Enable Powershell script execution on your system. See http://go.microsoft.com/fwlink/?LinkID=135170 for more information.
```

Comment:
>Enable Powershell script execution on your syst [](start = 4, length = 47)

I think an example would serve here better than the lin.
"Set-ExecutionPolicy -ExecutionPolicy RemoteSigned"